verilog/analysis/verilog_project_test.cc

```diff
@@ -592,5 +592,37 @@ TEST(VerilogProjectTest, ParseSourceFileList) {
               ElementsAre(".", "/an/include_dir1", "/an/include_dir2"));
 }
 
+TEST(VerilogProjectTest, ParseSourceFileListFromCommandline) {
+  std::vector<absl::string_view> cmdline;
+  // The first argument will be discarded.
+  cmdline.push_back("the_tool_name");
+  cmdline.push_back("+define+macro1=text1+macro2+macro3=text3");
+  cmdline.push_back("file1");
+  cmdline.push_back("+define+macro4");
+  cmdline.push_back("file2");
+  cmdline.push_back("+incdir+~/path/to/file1+path/to/file2");
+  cmdline.push_back("+incdir+./path/to/file3");
+  cmdline.push_back("+define+macro5");
```

Comment:
Maybe also a test with an equals sign but nothing following `+define+macro6=`

Reply:
That should be accepted?

Reply:
You can decide if it should be accepted or not at this stage. But if we have a test, then we document that.

Reply:
I added as an invalid syntax for now.